libs/server/src/lib/api/run-command.ts

```diff
@@ -46,10 +42,14 @@ function createExecutableCommand(
   cmds: string[]
 ) {
   return () => {
-    const commandRunning = spawn(program, normalizeCommands(cwd, cmds), {
-      cwd,
-      cols: 80
-    });
+    const commandRunning = require('node-pty-prebuilt').spawn(
```

Comment:
why did you move the `require`?

Reply:
Required by tslint now that this file is linted vs ng lint